src/main/scala/firrtl/passes/Checks.scala

```diff
@@ -51,7 +51,7 @@ object CheckHighForm extends Pass {
     s"$info: A single module must be named $name.")
   class NegArgException(info: Info, mname: String, op: String, value: Int) extends PassException(
     s"$info: [module $mname] Primop $op argument $value < 0.")
-  class LsbLargerThanMsb(info: Info, mname: String, op: String, lsb: Int, msb: Int) extends PassException(
+  class LsbLargerThanMsbException(info: Info, mname: String, op: String, lsb: Int, msb: Int) extends PassException(
```

Comment:
Nit: If the line limit is 100 characters you are over. Line break just before the "extends" keyword.

Reply:
I think 120 is the normal Scala-style limit, I've been moving toward 100 but the official limit remains 120